### ISSUE TITLE
[CAIA Intake] Benefits - Non-Disability Experience Team: VR&E (Chapter 31) Form Updates

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#75169]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Benefits - Non-Disability Experience 
- **OCTO product owner:** Emily Theis
- **Product name:** Veteran Readiness and Employment (VR&E) aka Chapter 31 (28-1900)
- **Product manager:** Laura Steele
- **Slack channel:** #benefits-non-disability
- **Dedicated content writer on your team (if you have one):** none
- **Dedicated a11y specialist on your team (if you have one):** none
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

The current online Veteran Readiness and Employment (VR&E aka Chapter 31) 28-1900 form is using an outdated version of the paper form. It's missing new fields and generating an old version of the form pdf. It also contains some accessibility issues that need to be addressed. The form intake API also needs to be migrated to a new RES API. Desired Outcome: Veterans submitting online form 28-1900 have a positive experience and can trust that their submissions include all the required information to ensure efficient processing.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
@fiorella-io has drafted the initial designs for the form updates in Figma, and we are reviewing those designs with VBA SMEs and our OCTO stakeholders.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** #74141 

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Form Flow in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1684161597392/5c1f8869cf5d03b792c8b82cf19b39dc26c486c2?sender=u0e8ac1d6d7681bb7e1b80558)
- [Figma Designs](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=276-19824&mode=design&t=uYBRL8zeUi3IrnbE-0)
- [VR&E Info Page](https://www.va.gov/careers-employment/vocational-rehabilitation/how-to-apply/)
- [VR&E Online Tool](https://www.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)
- [Form update initiative brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/education-careers/vocational-rehab/Initiative-Brief-VRE-Form-Updates.md)
- [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/education-careers/vocational-rehab/README.md)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [x] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [x] Initial Review - @coforma-terry and @strelichl
```

```[tasklist]
### CAIA a11y tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/63726
- [ ] #73590
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76966
```

```[tasklist]
### CAIA IA tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/77996
```

**Comment by steele-lm** [comment]
Copying @jstrothman

**Comment by jennymayoco** [comment]
Hi @steele-lm @fiorella-io @juliepedtke here is the[ IA spec](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/ch-31-form-updates.md) with notes on two entry points. I have also attached the [IA tracker](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77996) under `CAIA IA tasks`, where the IA spec can be found as well.

I noticed the current online form has the title tag `28-1900 Veteran Readiness | Veterans Affairs` - is this updated automatically when changes to the form are implemented?

**Comment by fiorella-io** [comment]
Hi @jennymayoco, where do you see that tag? [On the form page](https://www.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/introduction), this is the H1:
<img width="80%" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/91498500/f7c64aa3-38a9-464f-8284-120bc77cece4">

I don't see any tags.

**Comment by jennymayoco** [comment]
Hi @fiorella-io! I noticed it in page source for the current form intro page. I saw that on the proposed designs, the H1 is `Apply for Veteran Readiness and Employment` so I was wondering what the process is for title tags
 
<img width="493" alt="Title tag for form intro page" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/122128479/f40ebc4e-9569-4f33-93ec-95197f90adec">

**Comment by fiorella-io** [comment]
I'll check with our engineering team. I'm assuming this will be changed to reflect the new design copy.

**Comment by fiorella-io** [comment]
Hi @jennymayoco we'll be updating the title tag using this format `H1 (form #) | Veterans Affairs` so the new VR&E title will be `Apply for Veteran Readiness and Employment (VA Form 28-1900) | Veterans Affairs`

**Comment by jennymayoco** [comment]
thank you for sharing the updated title tag! Since the H1 in the new designs doesn't include the form #, is it possible to remove it from the title tag? That way the H1 matches and follows [title tag guidance](https://design.va.gov/content-style-guide/seo#title-tags). Let me know if there's any questions before today's midpoint! I'll be sitting in on the call as well.

**Comment by fiorella-io** [comment]
@jennymayoco Got it! we will remove the form number from the title tag
